sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,116 @@
+| `calibration_points`   | `tier4_calibration_msgs::msg::CalibrationPoints` | Publishes the tag points after calibration          |
+
+### Services
+
+| Name                    | Type                                                  | Description                                                                              |
+| ----------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration` | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocking until the calibration process finishes |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                          | Type                  | Default Value | Description                                                                                                                         |
+| --------------------------------------------- | --------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `calib_rate`                                  | `double`              |               | The rate at which the calibration callback is invoked. This controls the frequency of calibration updates.                          |
+| `base_frame`                                  | `std::string`         |               | The base_frame is used to compare the initial and calibrated values.                                                                |
+| `min_tag_size`                                | `double`              |               | The size of the AprilTag in meters                                                                                                  |
+| `max_tag_distance`                            | `double`              |               | Maximum allowable distance from the camera to the tags.                                                                             |
+| `max_allowed_homography_error`                | `double`              |               | Discard the AprilTag detection if the homography error is larger than this value.                                                   |
+| `use_receive_time`                            | `bool`                |               | Flag to determine whether to use the receive time instead of the header timestamps.                                                 |
+| `use_rectified_image`                         | `bool`                |               | Flag to determine whether rectified images should be used in the calibration process.                                               |
+| `calibration_crossvalidation_training_ratio`  | `double`              |               | The ratio of data used for training versus testing during the calibration's cross-validation process.                               |
```

Comment:
It is either validation or testing...

Reply:
@vividf 
Testing and validation have both different meanings yet depending on where you learned and from whom you learned, they have their meanings changed. Since you are already using cross-validation, you can not use testing

Reply:
Solved in https://github.com/tier4/CalibrationTools/pull/164/commits/f74fdb89b724f96b096c8f959a82787c77d706e8